app/Http/Controllers/Project/Planning/SearchStringController.php

```diff
@@ -25,13 +46,12 @@ public function index($id_project)
         $synonyms = $project->synonyms;
 
         $searchStrings = $this->getSearchStrings($id_project);
-        dd($searchStrings);
 
-        return view('planning.search-string', compact('project', 'terms', 'synonyms', 'searchStrings'));
+        return view('search-string', compact('project', 'terms', 'synonyms', 'searchStrings'));
```

Comment:
Se for o caso, mudar o `return view(...)`:

```suggestion
        return view('project.planning.search-string', compact('project', 'terms', 'synonyms', 'searchStrings'));
```